src/betterproto/__init__.py

```diff
@@ -563,7 +563,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         all_sentinel = True  # Keep track of whether every field was default
         group_current: Dict[str, str] = {}
 
-        for field in self.__raw_get(dataclasses._FIELDS).values():
```

Comment:
is this intentional?

Reply:
I was trying to fix the git tree so just added something to remove later